### ISSUE TITLE
ContentAfterOpenBracket and CloseBracketLine of FunctionCallSignature sniff ignores required spaces

### ISSUE THREAD

**Issue by withinboredom** [squizlabs/PHP_CodeSniffer#3126]
**Describe the bug**
It appears that `ContentAfterOpenBracket` and `CloseBracketLine` only deal with whether the open/closing parens are on a new line -- [based on their error message](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Standards/PEAR/Sniffs/Functions/FunctionCallSignatureSniff.php#L399-L422). Then, `requiredSpacesAfterOpen` and `requiredSpacesBeforeClose` appear to be ignored if those rules are effectively disabled.

**Code sample**
```php
<?php
test(                  'hello',
'world');
```

**Custom ruleset**
```xml
<?xml version="1.0"?>
<ruleset name="Disable specific rules">
	<rule ref="PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket">
		<severity>0</severity>
	</rule>
	<rule ref="PEAR.Functions.FunctionCallSignature.CloseBracketLine">
		<severity>0</severity>
	</rule>
</ruleset>
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See no error message displayed
```
. 1 / 1 (100%)


Time: 273ms; Memory: 8MB
```

**Expected behavior**
Expected to see an error message about spaces (or lack of) inside the function call.

**Versions (please complete the following information):**
 - OS: Linux (Debian)
 - PHP: 7.3.22
 - PHPCS: 3.5.6
 - Standard: PSR1, Zend, PSR2, PSR12, PEAR, WordPress-Core, WordPress, WordPress-Extra, WordPress-Docs
